### PR TITLE
fix: move help to top bar, move project settings to top bar

### PR DESCRIPTION
**Pull request by cabreraalex**
# Description
Why?
1. Makes settings in a consistent place across all views
2. Makes help a bit more discoverable, same in all views, not hacky depending on what is shown where.

Fix ZEN-411

image

### CODE REVIEW

File: frontend/src/lib/components/general/Header.svelte [hunk 1]
```diff
@@ -21,7 +21,7 @@
 	let linkCopied = false;
 </script>
 
-<div class="mx-6 my-5 flex h-8 items-center justify-between">
+<div class="mx-6 mb-4 mt-4 flex h-8 items-center justify-between">
```
Comment (Sparkier) [inline frontend/src/lib/components/general/Header.svelte:24]: nit: my-4

File: frontend/src/lib/components/general/UserButton.svelte [hunk 2]
```diff
@@ -1,34 +1,105 @@
...
+							showOptions = false;
+						}}
+					>
+						<Icon style="font-size: 20px;" class="material-icons">article</Icon>&nbsp;
+						<span class="whitespace-nowrap text-sm">Explore the Docs</span>
+					</button>
+				</Content>
+			</Paper>
+		</button>
 	{/if}
```
Comment (Sparkier) [inline frontend/src/lib/components/general/UserButton.svelte:104]: Should we move this in here? Will make it difficult if we ever want to add it to somewhere else (e.g. playground)

**Review by Sparkier — CHANGES_REQUESTED** [review]
I like the settings moving up. Am ok with help also moving but think it's much less prominent now. Might be fine since we're starting to link to docs from other places (e.g. reports).

If we keep it like this, I think we definitely should add it to the playground as well.

**Comment by cabreraalex** [comment]
Made it a component and added to playground!

File: frontend/src/lib/components/general/HelpButton.svelte [hunk 3]
```diff
@@ -1,29 +1,28 @@
...
+	import { Icon } from '@smui/icon-button';
 	import Paper, { Content } from '@smui/paper';
 	import { fade } from 'svelte/transition';
 
-	export let docsLink = 'https://zenoml.com/docs/intro';
-	export let optionsBelow = false;
+	export let showOptions = false;
 
-	let showOptions = false;
+	let docsLink = 'https://zenoml.com/docs/intro';
```
Comment (Sparkier) [inline frontend/src/lib/components/general/HelpButton.svelte:11]: Used this as export to deep-link to different parts of the docs previously but probably not necessary anymore.

**Review by Sparkier — APPROVED** [review]

**Pull request merged**